RecetarioExpo/src/data/recentData.js

```diff
@@ -0,0 +1,22 @@
+const images = [
+    require('../assets/images/spaguetti.jpg'),
+    require('../assets/images/vegetarian_pie.jpg'),
+    require('../assets/images/vegetarian_toast.jpg'),
+];
+
+const data = [
+    {
+        title: 'Spaguetti',
+        src: images[0],
+    },
+    {
+        title: 'Vegetarian Pie',
+        src: images[1],
+    },
+    {
+        title: 'Vegetarian Toast',
+        src: images[2],
+    },
+];
+
+export default data;
```

Comment:
As a rule of the Style Guide, always leave a blank line at the end of each file.